test/replication-luatest/gh_8433_replication_timeout_test.lua

```diff
@@ -0,0 +1,87 @@
+local luatest = require('luatest')
+local server = require('luatest.server')
+local replica_set = require('luatest.replica_set')
+
+local g = luatest.group('prevote_fail')
+
+local function get_term(server)
```

Comment:
There's a function `server:get_election_term()` in luatest.

Reply:
My bad. Fixed